docs/user-guide/petset.md

```diff
@@ -0,0 +1,370 @@
+---
+---
+
+* TOC
+{:toc}
+
+__Terminology__
+
+Throughout this doc you will see a few terms that are sometimes used interchangeably elsewhere, that might cause confusion. This section attempts to clarify them.
```

Comment:
It might be helpful to also specify if this document is a description of the petset system or a walk through. As I read it is hard for me to figure out if this is background or instructions I should follow to set up my own pet set. I am going to assume it is the latter for comments as I am reading it.